### PR TITLE
Fix translations messages_pt.yml

### PR DESCRIPTION
**Pull request by rafael59r2**

### CODE REVIEW

File: src/main/resources/messages/messages_pt.yml [hunk 1]
```diff
@@ -85,13 +85,13 @@ incomplete_email_settings: 'Erro: nem todas as definições necessarias para env
...
 show_no_email: '&2Você atualmente não tem um endereço de email associado a essa conta.'
-add_email: '&cPor favor adicione o seu email com : /email add seuEmail confirmarSeuEmail'
+add_email: '&cPor favor adicione o seu email com : /email add <seuEmail> <confirmarSeuEmail>'
 recovery_email: '&cPerdeu a sua password? Para a recuperar escreva /email recovery <seuEmail>'
 email_cooldown_error: '&cUm email já foi enviado recentemente.Por favor, espere %time antes de enviar novamente'
 
 # Captcha
 usage_captcha: '&cPrecisa digitar um captcha, escreva: /captcha <theCaptcha>'
-wrong_captcha: '&cCaptcha errado, por favor escreva: /captcha THE_CAPTCHA'
+wrong_captcha: '&cCaptcha errado, por favor escreva: /captcha <theCaptcha>'
```
Comment (ljacqu) [inline src/main/resources/messages/messages_pt.yml]: `THE_CAPTCHA` gets replaced with the captcha code—please revert.

**Comment by rafael59r2** [comment]
Fixed

**Comment by rafael59r2** [comment]
I fixed

**Comment by sgdc3** [comment]
thank you

**Pull request merged**